src/core/projectTeams/projectTeam.service.ts

```
@@ -57,6 +57,7 @@ import {
     ProjectTeamInvalidRecruitNumException,
     ProjectTeamInvalidTeamRoleException,
     ProjectTeamInvalidTeamStackException,
+    ProjectTeamInvalidUpdateMemberException,
```

ESLint

[error] 'ProjectTeamInvalidUpdateMemberException' is defined but never used.

(@typescript-eslint/no-unused-vars)